diff --git a/distribute.py b/distribute.py
index 5a311b1..97fb265 100755
--- a/distribute.py
+++ b/distribute.py
@@ -102,7 +102,7 @@ def main():
         args.version = ".".join(str(i) for i in renpy.version_tuple[:-1])
 
     try:
-        s = subprocess.check_output([ "git", "describe", "--tags", "--dirty", ]).decode("utf-8").strip()
+        s = os.environ.get("RENPY_GIT_DESCRIBE") or subprocess.check_output([ "git", "describe", "--tags", "--dirty", ]).decode("utf-8").strip()
         parts = s.strip().split("-")
         dirty = "dirty" in parts
 
