diff --git a/renpybuild/run.py b/renpybuild/run.py
index d97b51c..1237702 100644
--- a/renpybuild/run.py
+++ b/renpybuild/run.py
@@ -228,6 +228,7 @@ def build_environment(c):
         c.env("READELF", "{{ crossbin }}readelf")
 
         c.env("CFLAGS", "{{ CFLAGS }} -DSDL_MAIN_HANDLED")
+        c.env("LDFLAGS", "{{ LDFLAGS }} -Wl,-z,max-page-size=16384")
 
     elif (c.platform == "android") and (c.arch == "arm64_v8a"):
 
@@ -245,6 +246,7 @@ def build_environment(c):
         c.env("READELF", "{{ crossbin }}readelf")
 
         c.env("CFLAGS", "{{ CFLAGS }} -DSDL_MAIN_HANDLED")
+        c.env("LDFLAGS", "{{ LDFLAGS }} -Wl,-z,max-page-size=16384")
 
     elif (c.platform == "android") and (c.arch == "armeabi_v7a"):
 
@@ -262,6 +264,7 @@ def build_environment(c):
         c.env("READELF", "{{ crossbin }}readelf")
 
         c.env("CFLAGS", "{{ CFLAGS }} -DSDL_MAIN_HANDLED")
+        c.env("LDFLAGS", "{{ LDFLAGS }} -Wl,-z,max-page-size=16384")
 
     # elif (c.platform == "mac") and (c.arch == "x86_64"):
 
