diff --git a/tasks/pythonlib.py b/tasks/pythonlib.py
index 65169ac..a5566d6 100644
--- a/tasks/pythonlib.py
+++ b/tasks/pythonlib.py
@@ -255,7 +255,7 @@ def python3(c: Context):
                 dest.parent.mkdir(parents=True, exist_ok=True)
                 shutil.copy(fn, dest)
 
-    # used_rules.add("steamapi")
+    used_rules.add("steamapi")
 
     if rules - used_rules:
         if c.platform != "web":
