diff --git a/tasks/pythonlib.py b/tasks/pythonlib.py
index b10134d..2f409e8 100644
--- a/tasks/pythonlib.py
+++ b/tasks/pythonlib.py
@@ -204,8 +204,6 @@ six.pyo
 
 pefile.pyo
 ordlookup/
-
-steamapi.pyo
 """
 
 
@@ -236,7 +234,6 @@ def python2(c: Context):
         c.path("{{ install }}/lib/{{ pythonver }}/site-packages"),
         c.path("{{ pytmp }}/pyjnius"),
         c.path("{{ pytmp }}/pyobjus"),
-        c.path("{{ pytmp }}/steam"),
         ]
 
     dist = c.path("{{ distlib }}/{{ pythonver }}")
@@ -467,8 +464,6 @@ pyobjus/
 
 pefile
 ordlookup/
-
-steamapi
 """
 
 
@@ -488,7 +483,6 @@ def python3(c: Context):
         c.path("{{ install }}/lib/{{ pythonver }}/site-packages"),
         c.path("{{ pytmp }}/pyjnius"),
         c.path("{{ pytmp }}/pyobjus"),
-        c.path("{{ pytmp }}/steam"),
         ]
 
     dist = c.path("{{ distlib }}/{{ pythonver }}")
@@ -524,7 +518,7 @@ def python3(c: Context):
                 dest.parent.mkdir(parents=True, exist_ok=True)
                 shutil.copy(fn, dest)
 
-    # used_rules.add("steamapi")
+    used_rules.add("steamapi")
 
     if rules - used_rules:
         if c.platform != "web":
diff --git a/tasks/sysroot.py b/tasks/sysroot.py
index 85ed864..f3bf6f6 100644
--- a/tasks/sysroot.py
+++ b/tasks/sysroot.py
@@ -15,7 +15,6 @@ PACKAGES = [
     'libxxf86vm-dev',
     'libxss-dev',
     'libgl1-mesa-dev',
-    'libesd0-dev',
     'libdbus-1-dev',
     'libudev-dev',
     'libgl1-mesa-dev',
@@ -23,6 +22,7 @@ PACKAGES = [
     'libgles2-mesa-dev',
     'libglu1-mesa-dev',
     'libegl1-mesa-dev',
+    'libgbm-dev',
     'libibus-1.0-dev',
     'fcitx-libs-dev',
     'libsamplerate0-dev',
@@ -85,11 +85,11 @@ RASPI_PACKAGES = [
     'libxxf86vm-dev',
     'libxss-dev',
     'libgl1-mesa-dev',
-    'libesd0-dev',
     'libdbus-1-dev',
     'libudev-dev',
     'libgles2-mesa-dev',
     'libegl1-mesa-dev',
+    'libgbm-dev',
     'libibus-1.0-dev',
     'fcitx-libs-dev',
     'libsamplerate0-dev',
@@ -115,11 +115,11 @@ def install_linux_raspi(c: Context):
         --cache-dir={{ tmp }}/debs
         --variant=minbase
         --include={{ packages }}
-        --components=main,contrib,firmware,rpi
+        --components=main,contrib
         --arch armhf
         buster
         {{ sysroot }}
-        http://archive.raspbian.org/raspbian
+        http://archive.debian.org/debian
         """)
 
         c.run("""sudo cp /usr/bin/qemu-arm-static {{ sysroot }}/usr/bin """)
