diff --git a/tasks/pythonlib.py b/tasks/pythonlib.py
index 2ce0b76..1185224 100644
--- a/tasks/pythonlib.py
+++ b/tasks/pythonlib.py
@@ -201,8 +201,6 @@ six.pyo
 
 pefile.pyo
 ordlookup/
-
-steamapi.pyo
 """
 
 
@@ -233,7 +231,6 @@ def python2(c):
         c.path("{{ install }}/lib/{{ pythonver }}/site-packages"),
         c.path("{{ pytmp }}/pyjnius"),
         c.path("{{ pytmp }}/pyobjus"),
-        c.path("{{ pytmp }}/steam"),
         ]
 
     dist = c.path("{{ distlib }}/{{ pythonver }}")
@@ -503,8 +500,6 @@ pyobjus/
 
 pefile
 ordlookup/
-
-steamapi
 """
 
 
@@ -520,7 +515,6 @@ def python3(c):
         c.path("{{ install }}/lib/{{ pythonver }}/site-packages"),
         c.path("{{ pytmp }}/pyjnius"),
         c.path("{{ pytmp }}/pyobjus"),
-        c.path("{{ pytmp }}/steam"),
         ]
 
     dist = c.path("{{ distlib }}/{{ pythonver }}")
@@ -556,7 +550,7 @@ def python3(c):
                 dest.parent.mkdir(parents=True, exist_ok=True)
                 shutil.copy(fn, dest)
 
-    # used_rules.add("steamapi")
+    used_rules.add("steamapi")
 
     if rules - used_rules:
         raise Exception(f"Unused rules: {rules - used_rules}")
diff --git a/tasks/sysroot.py b/tasks/sysroot.py
index c230369..e91bb32 100644
--- a/tasks/sysroot.py
+++ b/tasks/sysroot.py
@@ -14,7 +14,6 @@ PACKAGES = [
     'libxxf86vm-dev',
     'libxss-dev',
     'libgl1-mesa-dev',
-    'libesd0-dev',
     'libdbus-1-dev',
     'libudev-dev',
     'libgl1-mesa-dev',
@@ -22,6 +21,7 @@ PACKAGES = [
     'libgles2-mesa-dev',
     'libglu1-mesa-dev',
     'libegl1-mesa-dev',
+    'libgbm-dev',
     'libibus-1.0-dev',
     'fcitx-libs-dev',
     'libsamplerate0-dev',
@@ -82,11 +82,11 @@ RASPI_PACKAGES = [
     'libxxf86vm-dev',
     'libxss-dev',
     'libgl1-mesa-dev',
-    'libesd0-dev',
     'libdbus-1-dev',
     'libudev-dev',
     'libgles2-mesa-dev',
     'libegl1-mesa-dev',
+    'libgbm-dev',
     'libibus-1.0-dev',
     'fcitx-libs-dev',
     'libsamplerate0-dev',
@@ -112,11 +112,11 @@ def install(c):
         --cache-dir={{ tmp }}/debs
         --variant=minbase
         --include={{ packages }}
-        --components=main,contrib,firmware,rpi
+        --components=main,contrib
         --arch armhf
         buster
         {{ sysroot }}
-        http://archive.raspbian.org/raspbian
+        http://archive.debian.org/debian
         """)
 
         c.run("""sudo cp /usr/bin/qemu-arm-static {{ sysroot }}/usr/bin """)

