diff --git a/tasks/toolchain.py b/tasks/toolchain.py
index b9b4d57..67df2d8 100644
--- a/tasks/toolchain.py
+++ b/tasks/toolchain.py
@@ -15,8 +15,8 @@ def build(c):
 
     c.clean()
 
-    c.env("CC", "ccache gcc-9")
-    c.env("CXX", "ccache g++-9")
+    c.env("CC", "ccache gcc")
+    c.env("CXX", "ccache g++")
 
     c.run("tar xaf {{ tars }}/binutils-{{ binutils_version }}.tar.gz")
     c.chdir("binutils-{{ binutils_version }}")
